CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -100,13 +100,15 @@ class SiPixelLorentzAnglePCLHarvester : public DQMEDHarvester {
   const double fitChi2Cut_;
   const int minHitsCut_;
   const std::string recordName_;
-  std::unique_ptr<TF1> f1;
+  std::unique_ptr<TF1> f1_;
   float width_;
   float theMagField_{0.f};
 
-  SiPixelLorentzAngleCalibrationHistograms hists;
-  const SiPixelLorentzAngle* currentLorentzAngle;
-  std::unique_ptr<TrackerTopology> theTrackerTopology;
+  static constexpr float inverseGeVtoTesla_ = 2.99792458e-3f;
```

Comment:
Wouldn't `inverseGeVperTesla` or `teslaToInverseGeV` be a more appropriate name for this conversion factor? (sorry for a late comment, catching up with stuff that was happening while being away on vacation)

Reply:
Won't bother now. Seems ok to me.